### DISCUSSION TITLE
New gamemode/mod idea

### DISCUSSION THREAD

**Discussion by StalkyBoi43815** [ppy/osu#27436] [Ideas]
I have come up with an interesting idea for a new mod or even gamemode, which I have yet to name. Basically, let's say there are, for example, grey and red hitobjects. You have to hit grey ones with Z and red ones with X (for example). This would greatly change the mapping scene. It would be a brand new skillset to learn. This change would allow for very **artistic** maps and more score increase mods. I guess this would have to be mapped on top of the vanilla map, or editor timeline could be used (?). Reading might be a problem, so there could maybe be a hud indicator or so. I know there are higher priorities at the moment, but this is just an idea.

**Comment by cihe13375** [comment]
This reminds me the idea of combining std with taiko (which is at least, once, tried (https://github.com/ppy/osu/pull/3320)).

**Reply by StalkyBoi43815** [reply]
> This reminds me the idea of combining std with taiko (which is at least, once, tried (https://github.com/ppy/osu/pull/3320)).

Sad it was abandoned, but at least there's hope (maybe)
